src/clients/patreon/patreonAgent.ts

```diff
@@ -1,18 +1,44 @@
-import patreonId from '../../models/patreonId';
-import { PatreonController, PatreonUserTokens } from './patreonController';
+import patreonRecord from '../../models/patreonRecord';
+import { PatreonController } from './patreonController';
 
-interface PatreonDetails {
+export interface PatronPublicDetails {
```

Comment:
Let's move this down to L43 next to PatreonAgent so it's clear it's part of that and not this IController